wiki/redirect.yaml

```
@@ -62,16 +62,16 @@
 "packs":                       "Beatmaps/Packs"
 "pack":                        "Beatmaps/Packs"
 
-"best_of_2009":                "Best_Of/2009"
-"best_of_2010":                "Best_Of/2010"
-"best_of_2011":                "Best_Of/2011"
-"best_of_2012":                "Best_Of/2012"
-"best_of_2013":                "Best_Of/2013"
-"best_of_2014":                "Best_Of/2014"
-"best_of_2015":                "Best_Of/2015"
-"best_of_2016":                "Best_Of/2016"
-
 "contest":                     "Contests"
+"best_of":                     "Contests#best-of"
+"best_of_2009":                "Contests#best-of"
+"best_of_2010":                "Contests#best-of"
+"best_of_2011":                "Contests#best-of"
+"best_of_2012":                "Contests#best-of"
+"best_of_2013":                "Contests#best-of"
+"best_of_2014":                "Contests#best-of"
```

```suggestion
"best_of_2014":                "Contests#2014.1"
```
same for 2015-2016

lol this is bound to break in the future if more years (for other contests) are added at any point. I'd also add a html comment at the beginning of the article that mentions this

---

This was actually why I just used `#best-of` instead of year number... Did we still want to do this anyways?

---

i think so, worst case is that it slips by all reviewers in the future and someone else will (hopefully) notice that it's broken on the site

chances of it slipping by are pretty low if you write a comment